packages/x-charts-pro/src/internals/plugins/useChartProZoom/useChartProZoom.types.ts

```diff
@@ -54,7 +55,11 @@ export interface UseChartProZoomPublicApi {
   setZoomData: (value: ZoomData[] | ((prev: ZoomData[]) => ZoomData[])) => void;
 }
 
-export interface UseChartProZoomInstance extends UseChartProZoomPublicApi {}
+export interface UseChartProZoomInstance extends UseChartProZoomPublicApi {
+  setZoomStart: (axisId: AxisId, value: number) => void;
+  setZoomEnd: (axisId: AxisId, value: number) => void;
+  moveZoomRange: (axisId: AxisId, by: number) => void;
```

Comment:
Some documentation would probably help in the future, which range is `value`, what is the scale of `by`